admin/app/controllers/authenticated/users/get.js

```diff
@@ -27,28 +36,36 @@ export default class GetController extends Controller {
   }
 
   @action
-  async reassignGarAuthenticationMethod(targetUserId) {
-    const authenticationMethodId = this._getGARauthenticationMethodId();
+  async reassignAuthenticationMethod({ targetUserId, identityProvider }) {
+    const authenticationMethodId = this._getUserAuthenticationMethodIdByIdentityProvider(identityProvider);
     try {
       await this.model.save({
-        adapterOptions: { reassignGarAuthenticationMethod: true, targetUserId, authenticationMethodId },
+        adapterOptions: {
+          reassignAuthenticationMethodToAnotherUser: true,
+          targetUserId,
+          authenticationMethodId,
+          identityProvider,
+        },
       });
       this.notifications.success(`La méthode de connexion a bien été déplacé vers l'utilisateur ${targetUserId}`);
       this.send('refreshModel');
```

Comment:
à voir si on peut faire du ember way ?

Reply:
Done ! Pour éviter le `send('refreshModel')` (et donc un appel API qui va chercher la terre entière) il suffit de dire qu'on souhaite destroy le record du store ember. (voir la doc ember associée : https://guides.emberjs.com/v4.1.0/models/creating-updating-and-deleting-records/)